vagrant.config.default.rb

```diff
@@ -0,0 +1,37 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# Vagrant machine configuration
+# ---------------------------------------------------------------------------------------------------------------------
+
+# Do not modify this file
+
+# To override these settings, make a separate copy of this file called `vagrant.config` in the same folder and make
+# your modifications in that file. This file is synced back to the repository and should only be changed if there is
+# a need to make a modification to the provisioning scripts
+
+@config_settings[@config_type] =
```

Comment:
Make a copy of this file in the same folder on your development machine called `vagrant.config.rb` and use this copy of the file to update the settings of your VM. The `vagrant.config.default.rb` is used as a fallback if you do not have your own local copy.

The `default` file is synced to the repository as an example and should not be changed unless the change is intended for all users. The local copy is included in the `.gitignore` so should not sync back.